cmd/corald/midware/auth.go

```diff
@@ -1,136 +0,0 @@
-package midware
```

Comment:
This code was moved into the `internal/platform/auth` package. No vendored deps were removed as none new were added.